runelite-client/src/test/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPluginTest.java

```diff
@@ -153,7 +153,7 @@ public void checkAnimationReset()
 		animationChanged.setActor(player);
 		plugin.onAnimationChanged(animationChanged);
 		plugin.onGameTick(new GameTick());
-		when(player.getAnimation()).thenReturn(AnimationID.LOOKING_INTO);
+		when(player.getAnimation()).thenReturn(AnimationID.DEATH);
```

Comment:
Rather than changing this footgun into another footgun, I'd like to see some constant defined in the test and used here instead of some other real known animation. eg. `private static final int UNKNOWN_ANIM = -2;`

Reply:
Makes sense :) I created a DUMMY_ANIM attribute to the class